lib/genesis_collector/disks.rb

```diff
@@ -38,6 +40,11 @@ def get_disk_info(cmd_params)
       shellout_with_timeout("smartctl -i #{cmd_params}", 5)
     end
 
+    def disk_uuid(drive)
+      blk_id = shellout_with_timeout('blkid', 5).split("\n").detect { |s| s.include? drive }
```

Comment:
Might want to memoize this, otherwise it's going to run it over and over for no reason